src/openrct2/world/Fountain.h

```diff
@@ -41,3 +41,13 @@ enum
     JUMPING_FOUNTAIN_TYPE_WATER,
     JUMPING_FOUNTAIN_TYPE_SNOW
 };
+
+namespace FOUNTAIN_FLAG
```

Comment:
Might be a good time to turn it into an `enum`?

Reply:
Sadly not possible to be done with flags. The preferred style is a namespace of constexprs